### PR TITLE
Add release notes

### PR DESCRIPTION
**Pull request by fhennig**
Inspiration: 
- k8s release page: https://kubernetes.io/releases/
- https://docs.docker.com/release-notes/
- https://docs.docker.com/engine/release-notes/

### CODE REVIEW

File: modules/ROOT/pages/release_notes.adoc [hunk 1]
```diff
@@ -0,0 +1,65 @@
+= Releases
+
+The Stackable platform consists of multiple operators that work together. Periodically a platform release is made, including all components of the platform at a specific version. We support the latest 3 versions.
```
Comment (razvan) [inline modules/ROOT/pages/release_notes.adoc:3]:
```suggestion
The Stackable platform consists of multiple operators that work together. Periodically a platform release is made, including all components of the platform at a specific version. The goal of the platform is to support the last three stable versions of each product.
```

File: modules/ROOT/pages/release_notes.adoc [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+== Release 22.06
+
+This is the first release of the platform, bringing operators for 12 products as well as stackablectl, the commandline tool to interact with stackable components. All operators spin up production ready product instances. Also there are some common features across all operators, such as monitoring, service discovery and configuration overrides. Find the <<Platform features>>, <<_stackablectl,stackablectl features>> and <<_operators>> below.
```
Comment (razvan) [inline modules/ROOT/pages/release_notes.adoc:7]:
```suggestion
This is the first release of the platform, bringing operators for 12 products as well as <<_stackablectl>>, the commandline tool to easily install data products in Kubernetes. Operators spin up production ready product applications. Also there are some common features across all operators, such as monitoring, service discovery and configuration overrides. Find the <<Platform features>>, <<_stackablectl,stackablectl features>> and <<_operators>> below.
```

File: modules/ROOT/pages/release_notes.adoc [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+=== Platform features
+
+==== Producting ready clusters easily set up
```
Comment (razvan) [inline modules/ROOT/pages/release_notes.adoc:11]:
```suggestion
==== Easily install production ready data applications
```

File: modules/ROOT/pages/release_notes.adoc [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+All operators spin up production ready product instances.
```
Comment (razvan) [inline modules/ROOT/pages/release_notes.adoc:13]: Using a familiar declarative approach, users can easily install data applications such as Apache Kafka or Trino across multiple cloud Kubernetes providers or on their own data centers. The installation process is fully automated while also providing the flexibility for the user to tune relevant aspects of each application.

File: modules/ROOT/pages/release_notes.adoc [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+==== Monitoring
+
+All products have monitoring with prometheus enabled.
+//
+xref:operators:monitoring.adoc[Learn more]
+
+==== Service discovery
+
+Across the platform you can use discovery ConfigMaps to connect to products.
```
Comment (razvan) [inline modules/ROOT/pages/release_notes.adoc:23]:
```suggestion
Products on the Stackable platform use service discovery to easily interconnect with each other.
```

File: modules/ROOT/pages/release_notes.adoc [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+==== Common S3 configuration
+
+Many products support connecting to S3 to load and/or store data. There is a common resource for S3 connections and buckets across all operators that can be reused.
+//
+xref:concepts:s3.adoc[Learn more]
+
+==== Roles and role groups
+
+Most products consist of multiple processes. All operators support configuring processes individually.
```
Comment (razvan) [inline modules/ROOT/pages/release_notes.adoc:39]: To support hybrid hardware clusters, the Stackable platform uses the concept of role groups. Services and applications can be configured to maximize hardware efficiency.

File: modules/ROOT/pages/release_notes.adoc (hunk 4 above)
Comment (fhennig) [reply modules/ROOT/pages/release_notes.adoc:13]: many thanks!

**Comment by fhennig** [comment]
This is the PR containing the latest operator versions: https://github.com/stackabletech/release/issues/7

**Review by soenkeliebau — APPROVED** [review]

**Pull request merged**